app/models/user.rb

```diff
@@ -2,4 +2,6 @@ class User < ActiveRecord::Base
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable,
          :confirmable
+
+  has_many :post
```

Comment:
Check whether `has_many :post` ought to be singular (`:post`) or plural (`:posts`).

The examples in the API docs for `has_many` may be useful: http://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html#method-i-has_many